bandit/blacklists/calls.py

```diff
@@ -343,6 +346,15 @@ def gen_blacklist():
         'Pickle library appears to be in use, possible security issue.'
```

Comment:
Add dill in this list

Reply:
what shall be the prefered wording ? `Pickle or Dill library appears to be in use ...` ?

Reply:
How about: "Pickle and modules that wrapper it can be unsafe when used with untrusted data."